packages/pigeon/lib/pigeon_lib.dart

```diff
@@ -282,6 +304,16 @@ class Pigeon {
       ));
     }
 
+    for (final ClassMirror enumMirror in enums) {
+      List<String> members = <String>[];
+      // The first 3 declarations and last two are trimmed as they are innate to
+      // enums in Dart and are not user defined values.
+      for (int i = 3; i < enumMirror.declarations.keys.length - 2; i++) {
+        members.add(MirrorSystem.getName(enumMirror.declarations.keys.toList()[i]));
```

Comment:
You are calling toList() here N times, it's worth moving that into a variable outside of the loop.